flytekit/core/map_task.py

```diff
@@ -1,72 +1,168 @@
-from typing import Any, Optional
+import os
```

Comment:
can we add module level docs, those get generated as docs and can be used in reference

Reply:
done